examples/objectdetection/src/CameraStream.tsx

```diff
@@ -132,7 +133,7 @@ export const CameraStream: React.FC<Props> = () => {
           ))}
         </Canvas>
         <Pressable style={styles.cameraSwitchButton} onPress={setActiveCamera}>
-          <Text style={styles.cameraSwitchButtonText}>Switch Camera</Text>
+          <Ionicons name="repeat-outline"></Ionicons>
```

Comment:
please address the linter error by turning this into an empty tag.

Reply:
@shackerman see above --^

Reply:
alright, fixed it!